fivetran_provider_async/operators.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Dict, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.utils.context import Context
+from fivetran_provider.operators.fivetran import FivetranOperator
+
+from fivetran_provider_async.triggers import FivetranTrigger
+
+
+class FivetranOperatorAsync(FivetranOperator):
+    """
+    `FivetranOperatorAsync` submits a Fivetran sync job , and polls for its status on the
+    airflow trigger.`FivetranOperatorAsync` requires that you specify the `connector_id` of
+    the sync job to start. You can find `connector_id` in the Settings page of the connector
+    you configured in the `Fivetran dashboard <https://fivetran.com/dashboard/connectors>`_.
+
+    :param fivetran_conn_id: `Conn ID` of the Connection to be used to configure
+        the hook.
+    :param connector_id: ID of the Fivetran connector to sync, found on the
+        Connector settings page in the Fivetran Dashboard.
+    :param poke_interval: Time in seconds that the job should wait in
```

Comment:
This has to be `poll_frequency` right?

Reply:
No, it would be `poke_interval` across trigger, sensor and operator.